packages/core/strapi/lib/commands/transfer/export.js

```diff
@@ -133,7 +142,8 @@ module.exports = async (opts) => {
     console.error('There was an error displaying the results of the transfer.');
   }
 
-  console.log(`${chalk.bold('Export process has been completed successfully!')}`);
+  // TODO: should the archive filename come before the exit text?
```

Comment:
@marcoautiero A successful export filename is the only time we ever displayed text *after* the "{Process} completed successfuly" text. Should we flip it around to keep it consistent, and always exit with the success message?